### PR TITLE
Dev: ui_node: redirect `node delete` to `cluster remove`

### LINKED ISSUE #891: `crm node delete` should adopt most of functionalities of `crm cluster remove` as mentioned in PR890
Probably this is low priority. Not likely many user use this interface.

> > `crm cluster remove tw3` works nicely so far.
> > Well, I expect the following alternative will do the same job. `crm node delete tw3`
> 
> Do you mean `crm cluster remove <node>` and `crm node delete` share the same code functions and execute the the process? Then we need another PR to do this

If the code can be reused directly, that will be great. From the behavior wise, `crm node delete <node>` should adjust_pcmk_delay_max here. Maybe, most of other behavior should be the same too. 

Or, create it as an alias of "crm cluster remove"

Or, deprecate `crm node delete`, and eventually drop it?

_Originally posted by @zzhou1 in https://github.com/ClusterLabs/crmsh/issues/890#issuecomment-979783182_

### PR DESCRIPTION
**Pull request by nicholasyang2022**
It will also adjust the related cluster configurations and clean up the
leftover (eg. stopping the cluster services) on the removed node.

```
# crm node delete 192.168.122.66
WARNING: `crm node delete` is deprecated and will very likely be dropped in the near future. It is auto-replaced as `crm cluster remove -c 192.168.122.66`.
Removing node "ha-1-2" from the cluster: Are you sure (y/n)? y
INFO: Stopping the corosync-qdevice.service on ha-1-2
INFO: Stopping the corosync.service on ha-1-2
INFO: Stopping the hawk.service on ha-1-2
INFO: Stopping the csync2.socket on ha-1-2
INFO: Removing the node ha-1-2
INFO: BEGIN Adjusting sbd related timeout values
WARNING: "stonith-timeout" in crm_config is set to 83, it was 71
INFO: END Adjusting sbd related timeout values
WARNING: "priority-fencing-delay" in crm_config is set to 60, it was 0
INFO: Propagating configuration changes across the remaining nodes
INFO: Done (log saved to /var/log/crmsh/crmsh.log)
```

Before this change, `node delete` only  remove node from cib:

```
# crm node delete ha-1-2
INFO: node ha-1-2 deleted
```

Close #891.

### CODE REVIEW

File: doc/crm.8.adoc [hunk 1]
```diff
@@ -2094,7 +2091,14 @@ Usage:
 delete <node>
 ...............
 
+.Deprecation note
+*****
+This command is deprecated in favor of `crm cluster remove [-F] -c <node>`,
+which will also stop cluster services and adjust related pacemaker configs.
+*****
```
Comment (liangxin1300) [inline doc/crm.8.adoc:2098]: @zzhou1 Could you please take a look at this man doc change?

File: crmsh/ui_node.py [hunk 2]
```diff
@@ -458,24 +518,21 @@ def _call_delnode(self, node):
...
-            return False
-        if not self._call_delnode(node):
-            return False
-        if utils.ext_cmd(self.node_delete % node) != 0 or \
-                utils.ext_cmd(self.node_delete_status % node) != 0:
-            logger.error("%s removed from membership, but not from CIB!", node)
-            return False
-        logger.info("node %s deleted", node)
-        return True
+        logger.warning('`crm node delete` is deprecated and will be dropped in the near future. It is auto-replaced as `crm cluster remove -c {}`.'.format(node))
```
Comment (liangxin1300) [inline crmsh/ui_node.py:530]: @zzhou1 And please review this warning expression

Comment (zzhou1) [reply crmsh/ui_node.py:530]: I'm okay with that. Or, if you want the the tone to be softer a little, I could suggest 

```suggestion
        logger.warning('`crm node delete` is deprecated and will very likely be dropped in the near future. It is auto-replaced as `crm cluster remove -c {}`.'.format(node))
```

File: doc/crm.8.adoc (hunk 1 above)
Comment (zzhou1) [reply doc/crm.8.adoc:2098]: How about this:
"""
This command is deprecated and in favor of `crm cluster remove [-F] -c <node>`,
which will adjust the related cluster configurations and clean up the leftover
(eg. stopping the cluster services) on the removed node.
"""

**Comment by liangxin1300** [comment]
@nicholasyang2022 Please add below changes similar like:
```
diff --git a/crmsh/bootstrap.py b/crmsh/bootstrap.py
index adde6b54..8a854976 100644
--- a/crmsh/bootstrap.py
+++ b/crmsh/bootstrap.py
@@ -2309,7 +2309,7 @@ def bootstrap_remove(context):
     if _context.cluster_node == utils.this_node():
         if not force_flag:
             utils.fatal("Removing self requires --force")
-        remove_self()
+        remove_self(force_flag)
     elif _context.cluster_node in xmlutil.listnodes():
         remove_node_from_cluster()
     else:
@@ -2318,14 +2318,14 @@ def bootstrap_remove(context):
     bootstrap_finished()
 
 
-def remove_self():
+def remove_self(force_flag):
     me = _context.cluster_node
     yes_to_all = _context.yes_to_all
     nodes = xmlutil.listnodes(include_remote_nodes=False)
     othernode = next((x for x in nodes if x != me), None)
     if othernode is not None:
         # remove from other node
-        cmd = "crm cluster remove{} -c {}".format(" -y" if yes_to_all else "", me)
+        cmd = "crm{} cluster remove{} -c {}".format(" -F" if force_flag else "", " -y" if yes_to_all else "", me)
         rc = utils.ext_cmd_nosudo("ssh{} {} {} '{}'".format("" if yes_to_all else " -t", SSH_OPTION, othernode, cmd))
         if rc != 0:
             utils.fatal("Failed to remove this node from {}".format(othernode))
```
Since run `crm -F node delete <local node>` will still prompted to confirm

**Review by liangxin1300 — APPROVED** [review]

**Pull request merged**